In [4]:
import pandas as pd

# Create data for the table
data = {
    'Random Forest': [12.61, 15.58, 242.66, .12],
    'WaterPrecipitationVSYear': [16.74, 22.01, 484.60, .01],
}

# Define row names (indices)
row_names = ['MAE', 'RMSE', 'MSE', 'R^2']

# Create the DataFrame
df = pd.DataFrame(data, index=row_names)

# Display the DataFrame
print(df)


      Random Forest  WaterPrecipitationVSYear
MAE           12.61                     16.74
RMSE          15.58                     22.01
MSE          242.66                    484.60
R^2            0.12                      0.01


Model Performance Metrics for Tehama:
Mean Absolute Error (MAE): 12.61
Root Mean Squared Error (RMSE): 15.58
R² Score: 0.12
Mean Squared Error (MSE): 242.66

Water precipitation versus the year. 
  Mean Absolute Error (MAE): 16.74
  Mean Squared Error (MSE): 484.60
  Root Mean Squared Error (RMSE): 22.01
  R² Score: 0.01
  